framework/src/play/src/main/java/play/mvc/Http.java

```
@@ -1875,6 +1875,27 @@ public JsonNode asJson() {
         }
 
         /**
+         * Parses the request to the expected class.
+         *
+         * @param <A> the type of the return value.
+         * @param clazz Expected Java value type.
+         * @return the return value.
+         */
+        public <A> A parseJson(Class<A> clazz) {
+            JsonNode asJson = asJson();
+            if (asJson == null) {
+                return null;
+
+            } else {
+                try {
+                    return Json.fromJson(asJson, clazz);
+                } catch(Exception e) {
```

What specific kinds of exceptions are you trying to catch here? I think it would be fine to let the exception bubble up.